### PR TITLE
Addon docs

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

**Comment by suutari-ai** [comment]
Can you rewrap the rst-text at 72-79 column or so? That would make it easier to read in text form (especially in the diffs).

File: doc/index.rst [hunk 1]
```diff
@@ -22,6 +22,7 @@ Developer documentation
    provides
    templates
    taxes
+   addons
```
Comment (suutari-ai) [inline doc/index.rst]: Addons and provides are somewhat connected. IMHO addons should be added before provides into that list. What do you think?

**Comment by akx** [comment]
Done, @suutari-ai

**Comment by suutari-ai** [comment]
Very nice doc! Thanks @akx! :D

**Comment by suutari-ai** [comment]
retest this please

**Pull request merged**